CMakeLists.txt

```diff
@@ -5,6 +5,19 @@ parseversion("src/rdkafka.h")
 
 project(RdKafka VERSION ${RDKAFKA_VERSION})
 
+set(CPACK_PACKAGE_VENDOR "Magnus Edenhill")
+set(CPACK_PACKAGE_CONTACT "info@edenhill.se")
+set(CPACK_PACKAGE_HOMEPAGE_URL https://github.com/edenhill/librdkafka)
```

Comment:
Should this have quotes around the URL?

Reply:
Not needed in cmake. But consistency is nice. Added